winml/test/common/SqueezeNetValidator.cpp

```diff
@@ -251,7 +251,7 @@ void ModelValidator::SqueezeNet(
         outputBindingStrategy, modelBinding, outputDataBindingName, expectedResultsTensor.Shape());
 
     // Evaluate the model
-    std::cout << "Calling EvaluateSync on instance" << instance << "\n";
+    std::cout << "Calling EvaluateSync on instance " << instance << "\n";
```

Comment:
does the output still have the correct spacing? (a space between "instance" and the value of instance)

Reply:
Yes, why wouldn't it?